### ISSUE TITLE
LUA Error: Button:SetPropagateMouseMotion()

### ISSUE THREAD

**Issue by Damari80** [Slothpala/RaidFrameSettings#87]
**Describe the bug**
Just a LUA Error

**To Reproduce**

**Expected behavior**
Just a LUA Error

**Have you disabled all other add-ons?**
No

**Screenshots**
8x [ADDON_ACTION_BLOCKED] AddOn 'RaidFrameSettings' tried to call the protected function 'Button:SetPropagateMouseMotion()'.
[string "@!BugGrabber/BugGrabber.lua"]:485: in function <!BugGrabber/BugGrabber.lua:485>
[string "=[C]"]: in function `SetPropagateMouseMotion'
[string "@RaidFrameSettings/Templates/AuraFrame.lua"]:173: in function `Enable'
[string "@RaidFrameSettings/Modules/BuffFrame.lua"]:89: in function `callback'
[string "@RaidFrameSettings/FrameEnvironment.lua"]:53: in function <RaidFrameSettings/FrameEnvironment.lua:36>
[string "@RaidFrameSettings/FrameEnvironment.lua"]:62: in function `func'
[string "@Blizzard_UnitFrame/Mainline/CompactRaidGroup.lua"]:29: in function `applyFunc'
[string "@Blizzard_CompactRaidFrames/Blizzard_CompactRaidFrameContainer.lua"]:130: in function `ApplyToFrames'
[string "@RaidFrameSettings/FrameEnvironment.lua"]:59: in function <RaidFrameSettings/FrameEnvironment.lua:58>
[string "=[C]"]: ?
[string "@RaidFrameSettings/Libs/AceBucket-3.0-4/AceBucket-3.0.lua"]:69: in function <...idFrameSettings/Libs/AceBucket-3.0/AceBucket-3.0.lua:67>
[string "@RaidFrameSettings/Libs/AceBucket-3.0-4/AceBucket-3.0.lua"]:83: in function `func'
[string "@Details/Libs/AceTimer-3.0-17/AceTimer-3.0.lua"]:57: in function <...ceDetails/Libs/AceTimer-3.0/AceTimer-3.0.lua:50>

Locals:
I was in BRD Raid

**Game Version**
11.0.5

**Additional context**
I had previously deactivated buff tooltips. I have also installed oiLvL addon

**Comment by Slothpala** [comment]
I don't think this is a problem on my end. The function should not be protected. It is important to set SetPropagateMouseMotion for help/harm mouseover stuff to work.
